very_good_core/hooks/test/pre_gen_test.dart

```diff
@@ -15,11 +16,15 @@ void main() {
     });
 
     test('populates variables', () {
+      // Fake the current year to ensure the test is deterministic.
+      final clock = Clock.fixed(DateTime(2024));
```

Comment:
I don't think this is how the Clock package is meant to be used. Notice that if you alter the date to be 2023 it will now fail. I believe you should also need to update the actual source code to rely on [Clock](https://pub.dev/documentation/clock/latest/clock/Clock-class.html) and then use [withClock](https://pub.dev/documentation/clock/latest/clock/withClock.html) on the test.

```suggestion
      final clock = Clock.fixed(DateTime(2023));
```

Reply:
Thank you for your feedback! I've updated the code to use Clock instead of DateTime.now(), which should ensure that the year is properly controlled across tests. I believe the issue is now resolved. Thanks again for your help!